### DISCUSSION TITLE
Busca estado / cidade

### DISCUSSION THREAD

**Discussion by GustavoTayer** [BrasilAPI/BrasilAPI#209] [Ideas]
Seria legal ter uma consulta que retorna a lista de cidades por estado selecionado. Ou um auto complete de cidade conforme o estado.

**Comment by LorhanSohaky** [comment]
Olá @GustavoTayer , já foram implementadas as rotas [/api/ibge/uf/v1/{code}](https://brasilapi.com.br/api/ibge/uf/v1/SP) e [/api/ibge/municipios/v1/{code}](https://brasilapi.com.br/api/ibge/municipios/v1/SP?providers=dados-abertos-br,gov,wikipedia). Acha que isso resolve?
